lepidopter-fh/setup-ooniprobe.sh

```diff
@@ -1,13 +1,35 @@
 #!/bin/bash
+set -ex
+
+TOR_DEB_REPO="http://deb.torproject.org/torproject.org"
+TOR_DEB_REPO_SRC_LIST="/etc/apt/sources.list.d/tor.list"
+TOR_REPO_GPG="A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89"
+
+# Add Torproject Debian repository
+apt-key adv --keyserver hkp://pool.sks-keyservers.net --recv-keys ${TOR_REPO_GPG}
+echo "deb ${TOR_DEB_REPO} ${DEB_RELEASE} main" > ${TOR_DEB_REPO_SRC_LIST}
+
 # Install golang-go
 apt-get update
-apt-get -y -t stretch install golang-go
+apt-get -y -t ${DEB_RELEASE}-backports install golang-go
 go version
 
-# Checkout ooniprobe git
-git clone https://github.com/TheTorProject/ooni-probe.git \
-/opt/ooni/ooni-probe-git/
-cd /opt/ooni/ooni-probe-git/
+# Build meek-client
+export GOPATH=$(mktemp -d)
+go get git.torproject.org/pluggable-transports/meek.git/meek-client
+cp $GOPATH/bin/meek-client /usr/local/bin/meek-client
+chmod +x /usr/local/bin/meek-client
+rm -rf $GOPATH
+# Install ooniprobe and pluggable transports dependencies
+apt-get -y install openssl libssl-dev libyaml-dev libsqlite3-dev libffi-dev \
+libpcap0.8-dev libgeoip-dev libdumbnet-dev tor tor-geoipdb python-dev \
+libgmp-dev python-pip
+# Install obfsproxy and fteproxy
+pip install obfsproxy fteproxy
+# Install obfs4proxy
+apt-get -y install -t stretch obfs4proxy
 # Install ooniprobe
-./scripts/install.sh
+# Remove previous version of pyasn1
+apt-get -y remove python-pyasn1
+pip install git+https://github.com/TheTorProject/ooni-probe.git
```

Comment:
Cool, I didn't know this was possible! Is this pulling from master? Is this a thing we want? Isn't it more reproducable to install using a specific tag?